src/app/client/src/app/modules/resource/modules/player/components/collection-player/collection-player.component.html

```diff
@@ -26,25 +26,37 @@
               [permission]="['CONTENT_CREATOR', 'CONTENT_REVIEWER']" (click)="copyContent(collectionTreeNodes.data)" class="sb-btn sb-btn-normal sb-btn-outline-primary mr-8">
               <i class="blue clone outline icon"></i> {{resourceService?.frmelmnts?.lbl?.copy}}
               </button>
             
-              <button type="button" appTelemetryInteract [telemetryInteractObject]="collectionInteractObject" [telemetryInteractEdata]="copyContentInteractEdata" *ngIf="permissionService.permissionAvailable && collectionTreeNodes.data.contentType === 'TextBook'" appPermission
+              <button type="button" appTelemetryInteract [telemetryInteractObject]="collectionInteractObject" [telemetryInteractEdata]="copyContentInteractEdata" *ngIf="permissionService.permissionAvailable && collectionTreeNodes.data.contentType === 'TextBook' && !isCopyAsCourseClicked" appPermission
               [permission]="['BOOK_CREATOR']" (click)="copyContent(collectionTreeNodes.data)" class="sb-btn sb-btn-normal sb-btn-outline-primary mr-8">
               <i class="blue clone outline icon"></i> {{resourceService?.frmelmnts?.lbl?.copy}}
               </button>
 
               <button type="button" appTelemetryInteract [telemetryInteractObject]="collectionInteractObject" [telemetryInteractEdata]="copyAsCourseInteractEdata" 
-              href="javascript:void(0)" *ngIf="permissionService.permissionAvailable && collectionTreeNodes.data.contentType === 'TextBook'" appPermission
-              [permission]="['CONTENT_CREATOR']" (click)="copyAsCourse(collectionTreeNodes.data)" class="sb-btn sb-btn-normal sb-btn-outline-primary">
+              href="javascript:void(0)" *ngIf="permissionService.permissionAvailable && collectionTreeNodes.data.contentType === 'TextBook' && !isCopyAsCourseClicked" appPermission
+              [permission]="['CONTENT_CREATOR']" (click)="copyAsCourse()" class="sb-btn sb-btn-normal sb-btn-outline-primary mr-8">
               <img src="assets/images/copy-clone.svg" class="mr-4 pt-2"> {{resourceService?.frmelmnts?.lbl?.copyAsCourse}}
               </button>
+
+              <button type="button" appTelemetryInteract [telemetryInteractObject]="collectionInteractObject" [telemetryInteractEdata]="cancelInteractEdata" 
+              href="javascript:void(0)" *ngIf="permissionService.permissionAvailable && collectionTreeNodes.data.contentType === 'TextBook' && isCopyAsCourseClicked" appPermission
+              [permission]="['CONTENT_CREATOR']" (click)="clearSelection()" class="sb-btn sb-btn-normal sb-btn-outline-primary mr-8">
+              {{resourceService?.frmelmnts?.btn?.cancel}}
+              </button>
+              
+              <button type="button"  appTelemetryInteract [telemetryInteractObject]="collectionInteractObject" [telemetryInteractEdata]="createCourseInteractEdata"
```

Comment:
Verify Object id should be current TextbookId. This is the must have, so that we can get to know which textbook is converting to course.